### DISCUSSION TITLE
Osu scorebar bottom half cut off after a certain point

### DISCUSSION THREAD

**Discussion by Xaexor** [ppy/osu#14223] [Q&A] [answered]
Hi, so there's this problem of the bottom of scorebar being cut off on all of my skins. Was there a new update that limits the dimensions of the scorebar now? This problem is on all the skins that have a larger scorebar

Tried all 3 releases, Stable, Beta and Cutting Edge. Same problem

![9ec7](https://user-images.githubusercontent.com/72856394/128994952-d6b9e001-4e25-40ff-8ad9-901a896079f4.png)
^Image of the scorebar being cut off

![svL2my4](https://user-images.githubusercontent.com/72856394/128995015-7ce6a3bd-f921-4702-b38e-e08a4b5cc59b.jpg)
^ What it looked like before the update

This happened to others that I asked too. Is there a chance that this was caused by the latest update (20210809.1). As I have tried multiple skins on another device running osu at an older version (20210520.2) with no issues to the healthbar

**Comment by mcendu** [comment, marked as answer]
This place is for questions regarding osu!lazer instead of stable I think?

**Reply by Xaexor** [reply]
Oh I didn't know. Mb.
Is there any place that I can raise questions/issues for stable?

**Reply by smoogipoo** [reply]
There's already an issue: https://github.com/ppy/osu-stable-issues/issues/809 Post your skin in there.
